### ISSUE TITLE
Order of sub elements is wrong sometimes.

### ISSUE THREAD

**Issue by Weeeqt** [WeakAuras/WeakAuras2#3430]
### Description

I created an aura with Action Button Glow and placed it below background (new feature from recent versions).  
The glow is supposed to be behind the icon (at background level), but sometimes it appears on top as if I didn't change the order.  
It can be reproduced if aura is supposed to be displayed and you /reload.

I thought that maybe Masque has something to do with it, but I tested it with and without Masque skin applied to this aura - it made no difference.

### WeakAuras Version

3.7.5

### World of Warcraft Flavor

Retail (Default)

### Tested with only WeakAuras

- [ ] Yes
- [ ] No

### Reproduction Steps

1. Create an icon aura with Action Button Glow placed under background level. Make sure it is always displayed.
2. Close WA options and /reload

### Screenshots

What I get on first load:
![World_of_Warcraft_2022-01-16_00-16-02](https://user-images.githubusercontent.com/7283432/149637981-397bf3db-d303-42a7-9b72-aecf6817228a.jpg)

What I expect:
![World_of_Warcraft_2022-01-16_00-14-36](https://user-images.githubusercontent.com/7283432/149637980-c233bf54-f47a-4a1e-83b1-7e223e2d9c3b.jpg)

### Export String

!WA:2!Lv1tVTrruC2yIqyXHw3qu6buxvHqTqP)XrnCcqztTBsXX2SEPTCaSND3z3zAwpZ2zM1joNO(akN9jo7ZCYFdi342ileFa6hHCajUXBM1jPLUsR9B)9EZ79MFV38gNM1gwlUw8jF6mAeN1JxiIWo(r8mU4jvGhX4ojjsSYzoIfr4IUCktfUtJ2bn8xeX5zX8dz9oKMJhC65F2iofteOifLZKB5lviHQQhLrvRpnM3pQqQ4djHjaIKS1mjVGf3phPiR4BLfULMmbm2cqMOe00uSq2(Zelf)Nzsc)WDY4mSK4PgNJ9rfcu9PMFzOHyz1xllcXJWmvxboHEKFVUnA16cWEfja48(7SDVG(9c22pW3IhUlgLPi(L(WRas7W8m0ySquiXTbuYezoolBVyz18RnePIi9JG0u1jhlqkUO23m)nqRD)xdRB)lbgCQjfTUGcUywmoesLayhi2TrRUn)HwtbRk3LvNhtLVOGbC5iCDeBC(vrw5Gs97ZJX)(7DgutWcgk7PafbKo6SdW48THqeP8rqzGmfXOdnsST0Fyt9AEmG3ofJK4EkbMLQiFWIyiRmw03qMcjg6hILtm2yYn998gIOStU(uzekd3eYPBwTQ(b66H2ULTRu()JRm5ChTsjIxf99)Gfah0xbvgzgsHjHwNCuLfxazIryzDxVM(ELgmUIEt99U0ituxzQXvwpt8rz5eKtTJwT24vNAXm(zHewbnLO2XGeMlWqh8mRTDlYK4Y8YVs5kmon)pCHNelrZz3kxWtHfjVJR4b3XnfEdHxe8kQdFdVHWlQ(TR6AFeyvHGbg7(fUxSy3p39wI6UFjaF7BB8Y)xzQrzQvz47Om0Om0Qe9okrgLiqPn(ywm8FOGRakYPS(ylzFTLQSFt0FutdVcLctnZqRqJNePm1kmzgCQWhNAoW(K)ErAg)WMc8llWSOXDF)7F36p0ZGn4uWFpgeS8kzIbZY3HfkfN1zewahuMzGBzBQ(45g5FC5iKPPNVu7Of9AciQgmRvpFPv2OhqOrhaNSLvSlQNzluk2IcWx1gdpUigliRRxBb4Oqu0bPcZ4ItwlVMcFKQVKGIV0XtTyMFe7VNVFh)E5x5nmRmXCCCQ86suCwIDAx9GoDZVMfdvaTNqVD0ZOXkI32WNNzvatsWXwWxTOCq55B6tT6Fb0yttgRxDwzvkaa9A3PDJssWyZYCUC1lZ5vNzXwooE3V9pVU5zdNkL7LeotLFLMc6XUFFbkgQKi3GG3AZFEASWIDiWyptGYN8SLclUWpM64Pb7U3oFxNFiO1ETBSKCEJX(NcAA433RtqqN9VCL9OhJVXI3kVxURhrL0Wm8Gtwt)to6Fo8rB3(Xn81dSSSgfw6jDKowJFLoXrNQxvt0uOr9f6dwvN9w7z9qntZD056xQfAPwnTmFGYJE0n0h6OpAqibBo3)kVmokoX2(ZGP6csOc6GyQ19hwKPOvfhdZ)Gb4R4hLHKY11)svpZ4YsbydzeMjSNimmJN52X6f045rPD(UnzBYKpT(0lAr0RoNYs4IYrSRNVbnLXf4o52RaByUwPHqWfYbM55MBt2d83gBr(nXrlzS81o)6ttVXJOseWCXeVJ58HoEMgpYmfpAu5e(nC6)V3KgN)jpcXGRbGPb7tTbWLN4gajcer36(hA7iNNiGBTGb9q)rLZklunnq2zU92XVrJ2lGiaWlZ4QWeswcnTQGYmregFSDf9VoyM5obAPjtopFhut6V5D)Q7(WAJ(RN)F)

**Comment by mrbuds** [comment]
I can't reproduce issue with <https://github.com/WeakAuras/WeakAuras2/pull/3427>

**Comment by InfusOnWoW** [comment]
I can't reproduce this either. Are you testing this aura standalone? We need a better description how to reproduce this.

I have also briefly looked at what we do in code and that looks fine to me too.

**Comment by Weeeqt** [comment]
I'm sorry for bumping this again, but I'm still running into this issue. I noticed that it happens during normal gameplay, not only in options.
I have many WAs (like really a lot of them), and I can easily reproduce this issue by adding the aura from the import string I provided in the ticket. Perhaps it is somehow related to the amount of auras I'm using?
I backed up all my auras and removed all of them just to test this weak aura alone and you are right - I cannot reproduce this as well.

I made two short clips from my stream demonstrating the issue:
https://clips.twitch.tv/UnusualAbstemiousFalconSpicyBoy-vh2Ltez7k82-kAxZ
https://clips.twitch.tv/SpotlessPlayfulPrariedogKappa-iZeoFJXz3ed2iOI_

1. In the first clip the arena match just started and enemy shadow priest applied Devouring Plague on me. I have WA for this and because it's first time this WA is displayed it has red glow on the foreground instead of background. This WA has similar (nearly identical) display settings to the one I provided in the import string.
2. In the second clip we can observe another application of Devouring Plague in the same arena match. this time the glow is displayed as expected in the background.

It seems that this issue is somewhat random. Sometimes it works right away as expected, sometimes not.

**Comment by InfusOnWoW** [comment]
If you have other auras, then your reproduction steps are not complete. And you should really give us more information on how to reproduce it. And crucially: TEST the reproduction steps before you tell us how to reproduce it. If that requires your settings and then importing that aura, then we need your settings. If importing the aura does not reproduce it, then we need a better description on how to reproduce it.

We have a much better chance of fixing bugs, where we can reproduce it. The clip doesn't help in that regard, it's not even the same aura.

**Comment by Weeeqt** [comment]
I understand that. I figured out how to reproduce it even with no other auras at all.
The aura from provided import string must be placed in a dynamic group.

Here is new import with a group which I tested it on and being able to reproduce it all the time:
!WA:2!Lzv3UnXruyC2MsZcTqmbxGsBxLwrj8te1Oqf9hQ8MytsXeBwB(R34m7UJ3DkRNzzMztIZnvyHQ4Yk)i4R7v(nO5jyKvvFaYJaxuPExpZSoafI0U7mNZzMZpFNVJtHhuu4D9L)MLxPyVIHfdFXYUrC2oUR14HBEGpJhI5UOGNeYzPPN0nHS3EiEOtBglrssNGYKXmEJujHrf28(n62vGLfgJObG8Mmcv6VA1nBx1Bu3SKKvj8Ge8wEIuuaEwokWCUB4jKiU02LqjsB)UWhrS9ajNefH5IZDb(0L)2OqSFw3UT7NI5RxTEZA3VURe24HY4OYdeP4KKncf2hiY8XBJPYwG1KDh3z1kTA3Pv7kETDZaN4NMG6J5VYSMCmyMxRMvRx3ZiXFDmkrg7rr9Wc7Hz0PHG94awswpAlDgy9si)WCkk5bqCcPcAOaN01K2LB3OPhkHervZnGZKiDQwye4rpCKPA5MWqH1CfK9WL86Lb1tBxidckPUKTxqcsi0R8LOeiE0lN4pfiwLLW4)KLLvtRRT8kdI4KqDjzrV14ikPhsIJ9ebW5S8ZrW4r5FRggHtNV1tZqCStnaqCEymrIhXnHK(o2pSpKYKaOhilD6PAbr4SUcgx6szu8WCFah4gQpSMAb1n3hJe4wsoMgjJp6Kqam0Q7OHgUahWOHIbAB0EqDt3Eic1Cq13Du13R(b1TGLN4TL4ZrHKmXZp(GmbUoPhrgpeXdQB8YZlDaCTqFdvwdAJyCFVkRTX9B5LOn8y7Nh55TJZnoF3guyJ1ijly7C46mf68Vlsch1glKo3wNXd11YhscLXhBCxoG9qwHKiRxM3qxtlsNe(Tw1RA1nbGDN8oHYzKWXp(o3BVkRFVG(RW8HORljYMV7HuI8yih6kuOG1iDzHKtCgtODz88IQnhYkDRwHIbN7fF5icyxlwgpaF(JOoAb1hOMBsaq)cz7qBTdjfV1(hUvdVXU6dSL6KBEb18)ZirmBNvtautedL3tNEkWjbXDcyqhDdaDqqPR4TuFI687R5qggebyqQpv9zQZQo3qT0CsahGHnHvXQs2Qp3E8BCtfV2bGY7(AbBP(y1zufvNs5ypoKi(LmkW13gxgr7NopYSUDoJ6USq8FCe1IOx(emoTIMbi90vIy1xOMdy8cKu3qb(9RU0FpjkHTtno(PzyAq)MV31wU8kUAzBTpeb3gwyQWXd0Y0i1e)mPKrBaqoW5hPfN3bD6X61pEk8mm6WJQ5vwQf4atvlZy1JEtRAzywMaPDmj4jqTvyz0uNaRN34d3CExj1ctGlstC1mkAOFmMeflFM6IDhcbChkmTHhRUQMFRwY8(kM3xw)MVhGCWaTzuRO(z17Bp4qKElvBygXRdxVcQ7z7UhJ1RGlmtgQDRpT5EOP5Ucub92r3x)m1DSu1v3fMnLgJSunghe14oxNEDQ4bL5eQMAaL6JdCBB1jGrZ6OM7eKjKSEJm76KIKXZmiK1XSnw9rLgcBYnjw9TMXcapwtNpKD)JV4S(M41ZY3mAAxR89vM(92wM6HrxS(y(MHM4ct0IbwivKOBcgC40Lz03)qJ9ggzU79t5yaPgzYUMzjcS29tE15RbnIZKhb9Tu)AHHgNBAueGrASX0fuS)Sf3DwtizSi(1xHzi2VNFq91L(NoWFDnD4m6ft5SiigexXH)1xXjcE8Hhe8Wld7HhF4bvEjBhZFCm0EtbJDUSZRoSZLCUiVSZvbXlTK(wEBLrALrgL(VJsFTsFJs07OePvIaLg)JPHWxaTYRK68zrB7CiX1spu()dKAYS(hp3iqJa3Vu6zGFHJXXt)xaQQ)TZQCoJl2kDHd7wBJ3vUgrG8tWHXf3(VE0)n
